api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/util/nameConventions.kt

```diff
@@ -32,13 +32,13 @@ internal fun depthwiseConv2dBiasVarName(name: String) = name + "_" + "depthwise_
 internal fun depthwiseConv2dKernelVarName(name: String) = name + "_" + "depthwise_conv2d_kernel"
 
 /** Default SeparableConv2d bias variable name in TensorFlow graph, based on variable's name. */
-internal fun separableConv2dBiasVarName(name: String) = name + "_" + "separable_conv2d_bias"
+internal fun separableConvBiasVarName(name: String, dim: Int) = name + "_" + "separable_conv2d_bias"
```

Comment:
`dim` parameter is not used at this moment.. Are you going to create separate variable names depending on dims? Isn't it?